service/controller/resource/tccpn/create.go

```diff
@@ -322,6 +322,7 @@ func newSecurityGroupsParams(ctx context.Context, cr infrastructurev1alpha2.AWSC
 			MasterSecurityGroupRules:   masterRules,
 			EtcdELBSecurityGroupName:   key.SecurityGroupName(&cfg.CustomObject, "etcd-elb"),
 			EtcdELBSecurityGroupRules:  getEtcdRules(cfg.CustomObject, cfg.ControlPlaneVPCCidr),
+			VPCID:                      cc.Status.ControlPlane.VPC.ID,
```

Comment:
this lines up with the template, not sure if it is still required here though.

Reply:
I thought this needs the Tenant Cluster VPC ID, not the Control Plane one.

Reply:
made the change here, let me know if the pr is good to merge.